dpytools/config/README.md

```diff
@@ -0,0 +1,159 @@
+# dpytools: Config
+
+Configuration class to enforce and police python application settings.
+
+## Usage
+
+The `from_env()` method is used to define a Config object and populate it with 
+a dictionary as input. The attributes of the created Config object are generated 
+from the contents of the dictionary. See below example.
+
+## Basic Example
+
+Say we want to instantiate a config object, setting our configuration with an environment variable.
+The environment variable will be called `MY_SERVER_ENDPOINT` and we want its value to be a URL.
+
+```python
+# The environment variable we want to use for our config. The value is a string of a URL.
+os.environ["MY_SERVER_ENDPOINT"] = "http://my-url.com"
+```
+
+When using the `from_env()` method to instantiate the config, we want to look up the  environment 
+variable's value and to also make sure that value is a string that looks like a URL, so let's say
+it should begin with "http://".
+
+The code below shows how this instantiation would work using the `from_env()` method.
+
+```python
+from dpytools import Config
```

Comment:
Import statement needs to be `from dpytools.config.config import Config` (and in other examples)